schduler/worker.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+import redis
+from rq import Worker, Queue, Connection
+
+listen = ['high', 'default', 'low']
+
+redis_url = os.getenv('REDISTOGO_URL', 'redis://localhost:6379')
```

Comment:
these are necessary setup for reds and rq. you could find the detail in here https://devcenter.heroku.com/articles/python-rq